package.json

```diff
@@ -31,7 +31,7 @@
     "lint-fix": "eslint --fix *.js webpack",
     "stylelint": "stylelint webpack/**/*.scss",
     "stylelint-fix": "stylelint webpack/**/*.scss --fix",
-    "test": "jest --no-cache"
+    "test": "jest --no-cache webpack"
```

Comment:
Very nice.

Maybe a bit off topic
Is there a reason we are not using tfm-test?
https://github.com/theforeman/foreman_ansible/blob/master/package.json#L29
https://github.com/theforeman/foreman_puppet/blob/master/package.json#L8
https://github.com/theforeman/foreman_remote_execution/blob/master/package.json#L8

Reply:
No specific reason. The `tfm-*` commands are used to supply a generic configuration to the underlying command (jest, eslint) unless a local one is already present.
This makes most sense for eslint (`tfm-lint`) IMHO, to ensure a shared code-style. We are not using tfm-lint currently, because there were some issues. We might want to in the future.

As for `tfm-test`: since we already have our own jest.config.js, so IMHO using `tfm-test` should not make a difference. We might want to take another look in the future :sweat_smile: